tests/functional/i13n.spec.js

```diff
@@ -1,12 +1,12 @@
 /* global describe, it, document, expect */
 
 describe('React I13n test', () => {
-  it('should init correctly and give the rootI13nNode the correct default model value', () => {
+  it.only('should init correctly and give the rootI13nNode the correct default model value', () => {
     const result = window._reactI13nInstance.getRootI13nNode().getMergedModel();
     expect(result).to.eql({ sec: 'default-section-name', page: 'test-page' });
   });
 
-  it('should fire a pageview', () => {
+  it.only('should fire a pageview', () => {
```

Comment:
will enable test one by one to make sure all cases are covered (except if we have breaking changes)